tests/features/community_content/notification.post_moderated.discussion.feature

```diff
@@ -86,7 +86,7 @@ Feature: Notification test for the discussion transitions on a post moderated pa
     Then the following email should have been sent:
       | recipient | CC member                                                                                                                                                        |
       | subject   | Joinup: Content has been updated                                                                                                                                 |
-      | body      | the Facilitator, CC Facilitator has approved your request to publish the discussion - "CC notify post approve proposed" in the collection: "CC post collection". |
+      | body      | the Facilitator, CC Facilitator has approved your request of publication of the discussion - "CC notify post approve proposed" in the collection: "CC post collection". |
```

Comment:
For the first part, the
> the
Comes after the "Dear <firstname>," which is not mentioned here, and this ends in a comma. The next paragraph does not start with a capital letter as it is considered an extend of this phrase.
> Facilitator,
This is a role name. That depends on the definition of the role but you can argue with Andrea on that.
> CC Facilitator
Indeed a comma could be added there. The previous comma was not misplaced as normally you would have something like "The facilitator, Ilias Dimopoulos, has done something". So the first comma is also applicable.
> has approved your request of publication of the discussion
Surely the 'has approved your request to publish the discussion' sounds better but I do not know if this is wrong grammar (maybe I am wrong).

For the last two parts, hyphens and double colons are used in sentences like this. Again, it is not a best practice but I did not QA the texts. Please open follow ups for this or discuss directly with Andrea. The notification system is the scope here.